generator/lib/behavior/i18n/I18nBehaviorObjectBuilderModifier.php

```diff
@@ -183,7 +183,7 @@ protected function addTranslatedColumnSetter(Column $column)
             $objectBuilder->addMutatorOpenOpen($functionStatement, $column);
         }
         $comment = preg_replace('/^\t/m', '', $comment);
-        $comment = str_replace('@return     ' . $i18nTablePhpName, '@return     ' . $tablePhpName, $comment);
+        $comment = str_replace('@return ' . $i18nTablePhpName, '@return ' . $tablePhpName, $comment);
         $functionStatement = preg_replace('/^\t/m', '', $functionStatement);
```

Comment:
Are you sure it was not intentional? For example, you have a lot of code in generator/lib that look like:
`_* @return_____$queryClassName`

(I've replaced spaces by _ so they are not dropped)